### PR TITLE
feat: add polygon logs processing - Closes #issue-8

### LINKED ISSUE #8: Add Polygon logs processing

### PR DESCRIPTION
**Pull request by IlluvatarEru**
Signed-off-by: Arthurim <arthurbdauphine@gmail.com>

## What does this PR do?

This PR adds logs processing for Polygon.

## Related issue

Closes #8 

## Testing
Using a Pocket Network RPC URL for Polygon, I ran several instances of `inspect-many` (eg: ./mev inspect-many 34500000 34500100`) and it correctly detected mev transactions.

## Checklist before merging
- [x] Read the [contributing guide](https://github.com/flashbots/mev-inspect-py/blob/main/CONTRIBUTING.md)
- [x] Installed and ran pre-commit hooks
- [x] All tests pass with `./mev test`

### CODE REVIEW

File: mev_inspect/inspect_block.py [hunk 1]
```diff
@@ -1,236 +1,89 @@
...
+                        str(liq.debt_token_address).replace(TRAILING_ZEROS, ""),
+                        liq.debt_purchase_amount,
+                        str(liq.received_amount).replace(TRAILING_ZEROS, ""),
+                        liq.received_amount,
+                    ]
+                )
+
+    if count > 0:
+        print("writing profits of {0} mev transactions".format(count))
+        # @TODO: Write profits to DB
```
Comment (0xpanoramix) [inline mev_inspect/inspect_block.py:86]: Nice, I'm just marking this for reference

**Review by 0xpanoramix — CHANGES_REQUESTED** [review]

File: mev_inspect/queue/tasks.py [hunk 2]
```diff
@@ -20,16 +19,12 @@ def inspect_many_blocks_task(
     after_block: int,
     before_block: int,
 ):
-    with _session_scope(DbMiddleware.get_inspect_sessionmaker()) as inspect_db_session:
-        with _session_scope(DbMiddleware.get_trace_sessionmaker()) as trace_db_session:
```
[the remark this answers is not in the document]
Comment (IlluvatarEru) [reply mev_inspect/queue/tasks.py:24]: `trace_db_session` and `inspect_db_session` are not used anymore in `inspect_many_blocks`

[the remark this answers is not in the document]
Comment (0xpanoramix) [reply mev_inspect/queue/tasks.py:24]: I think we will need to keep the inspect_db_session as we will use it to store data in the Inspect DB, but feel free to remove the trace db session and then merge !

[the remark this answers is not in the document]
Comment (IlluvatarEru) [reply mev_inspect/queue/tasks.py]: Alright, added `inspect_db_session` back

**Review by 0xpanoramix — APPROVED** [review]

**Pull request merged**